src/Library/DBusIPCServerPrivate.cpp

```diff
@@ -339,6 +527,30 @@ namespace usbguard
       with_interface_string.c_str());
     return builder;
   }
+
+  void DBusIPCServerPrivate::addAllowedUID(uid_t uid, const IPCServer::AccessControl& ac)
```

Comment:
What is the current behavior without these functions implemented? If it is expected for them to be empty, could you add a comment to make that clear? Thanks

Reply:
I added a comment.